src/pages/evm/wallet/WalletPage.vue

```diff
@@ -1,29 +1,188 @@
 <script lang="ts">
 import { defineComponent } from 'vue';
+
+import { EvmTokenInfo } from 'src/antelope/types';
```

Comment:
following the other comment, I would suggest using EvmToken instead of creating a new interface. This change will need to refactor the rest of the file.

```suggestion
import { EvmToken } from 'src/antelope/types';
```